### ISSUE TITLE
Missing letters in formatted file

### ISSUE THREAD

**Issue by xiamx** [vvakame/typescript-formatter#74]
Hi,

We use typescript-formatter as an essential building block for [Schemats](https://github.com/SweetIQ/schemats/). We recently discovered that the formatted output from typescript-formatter would miss some letters in the 3rd line of the output. Some discussion around this happened on https://github.com/SweetIQ/schemats/issues/37

I've created an example isolating the issue.

Input file `ab.ts`
```typescript

            /**
             * AUTO-GENERATED FILE @ 2016-12-07 13:17:46 - DO NOT EDIT!
             *
             * This file was generated with schemats node package:
             * $ schemats generate -c postgres://username:password@localhost/test -t users -o ./test/osm.ts
             *
             * Re-run the command above.
             */

    
        export namespace cta_situationcompte_scoFields {
        export type sco_id = number;
export type sco_total_debit = number;
export type sco_total_credit = number;
export type sco_nb_ecritures = number;
export type pst_id = number;
export type cpt_id = number;

        }
    
        
```

Output from tsfmt

```typescript
/**
 * AUTO-GENERATED FILE @ 2016-12-07 13:17:46 - DO NOT EDIT!
 *
 * This file was generated with schemats node package:
 * $ schemats generate -c postgres://username:password@localhost/test -t users -o ./test/osm.ts
 *
 * Re-run the command above.
 */


export namespace cta_situationcompte_scoFields {
    export type sco_id = number;
    ex type sco_total_debit = number;
    export type sco_total_credit = number;
    export type sco_nb_ecritures = number;
    export type pst_id = number;
    export type cpt_id = number;

}
```
On the 3rd line of the output there's missing letter for the token `export`

Please help us :)

**Comment by xiamx** [comment]
A fix is proposed in #75

**Comment by vvakame** [comment]
thanks!
I just published v4.1.1.

**Comment by xiamx** [comment]
Thanks for the quick release! :tada:
